pkg/controller/combo_controller.go

```diff
@@ -2,36 +2,191 @@ package controller
 
 import (
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
 	"github.com/go-logr/logr"
+	"github.com/operator-framework/combo/api/v1alpha1"
+	combinationPkg "github.com/operator-framework/combo/pkg/combination"
+	templatePkg "github.com/operator-framework/combo/pkg/template"
```

Comment:
nit(convention): self references should go into a third import group at the bottom

Reply:
That's clean, I agree.